cedars/app/db.py

```diff
@@ -1343,8 +1372,12 @@ def data_generator():
             notes = get_all_notes(patient_id)
             reviewed_notes = [note for note in get_patient_notes(patient_id, reviewed=True)]
             note_details = []
+            reviewers = []
             for note in notes:
                 note_id = note["text_id"]
+                reviewed_by = get_note_reviewer(note_id)
```

Comment:
Just get the last entity (cedars/pines/users) who touched(adjucdicated, reviewed etc) the record